sklearn/linear_model/tests/test_logistic.py

```diff
@@ -443,11 +444,12 @@ def test_ovr_multinomial_iris():
     n_samples, n_features = train.shape
 
     # Use pre-defined fold as folds generated for different y
-    cv = StratifiedKFold(target, 3)
-    clf = LogisticRegressionCV(cv=cv)
+    cv = StratifiedKFold(3)
+    predefined_folds = list(cv.split(train, target))
```

Comment:
I recollect why I did this now! Adding to @vene's explanation..

1st clf uses the un altered dataset... and uses stratified cv (stratified based on the separated class setup).

2nd clf (clf1) conflates classes 0 and 1 and uses this altered dataset to train and test **but** stratification is based on the separated class setup. I guess the motive behind such a stratification is to get equal no. of samples from (the previously separated classes 0 and 1). Should I leave this as such?